feat(curriculum): add workshop python functions (#68279)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/workshop-kitchen-inventory-tracker/6a3e2b9b7eeca084dd4fb857.md

---
id: 6a3e2b9b7eeca084dd4fb857
title: Step 1
challengeType: 20
dashedName: step-1
---

# --description--

> Welcome to the Python Functions and Scope workshop! You are going to learn how functions work using a kitchen theme.

Reviewer: ```suggestion
Welcome to the kitchen! Just like a chef relies on recipes to prepare a meal, you will rely on functions to build software. In this workshop, you'll learn how to write reusable Python functions and master 'scope', understanding exactly which ingredients (variables) are available in your code at any given time.
```

> Welcome to the Python Functions and Scope workshop! We are going to learn about how functions work using a kitchen theme.

Reviewer 2: please don't use second person, workshops should use "you", not "we" or "us"

> Welcome to the Python Functions and Scope workshop! You are going to learn about how functions work using a kitchen theme.

Reviewer: ```suggestion
Welcome to the Python Functions and Scope workshop! You are going to learn how functions work using a kitchen theme.
```

> To get started, You should define some basic data for your kitchen inventory. Create three variables: `eggs`, `flour`, and `sugar`, and assign them the values `1`, `2`, and `3` respectively.

Reviewer 2: ```suggestion
To get started, you should define some basic data for your kitchen inventory. Create three variables: `eggs`, `flour`, and `sugar`, and assign them the values `1`, `2`, and `3` respectively.
```
